net.sonic_pi.SonicPi.json

```diff
@@ -205,16 +209,12 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://github.com/sonic-pi-net/sonic-pi/archive/v3.3.1.tar.gz",
-                    "sha256": "6aab8e22daff1832b7af8221d7d124e4ae0adeaf1c726d8f62a8d3d57bd27d9c"
-                },
-                {
-                    "type": "patch",
-                    "path": "sonic-pi-fixes.patch"
+                    "url": "https://github.com/sonic-pi-net/sonic-pi/archive/refs/tags/v4.2.0.tar.gz",
+                    "sha256": "22376b5acf528582c6b8f43a6c53df368775ac4cc7b1136fdfc073c05d2ad56d"
                 },
                 {
                     "type": "patch",
-                    "path": "sonic-pi-sc-path.patch"
+                    "path": "sonic-pi-no-libstdc++.patch"
```

Comment:
From what I could gather, the patches that were previously here are no longer needed.

I needed to add a new one though (see comment below).